### DISCUSSION TITLE
When do you guys reckon it'll be updated to 1.19.2?

### DISCUSSION THREAD

**Discussion by andretbunngirl** [DynamicTreesTeam/DynamicTrees#729] [Q&A]
I've returned to Minecraft mods after a few years, and I've been using the most updated version of Minecraft, and returning to the old mods I had alot of fun with as a kid, though this one is quite outdated. Do you guys think you'll be updating this to 1.19+ soon? I would really love to see it! Also apologies if this isn't the right place to ask, the comments seem to be turned off on the CurseForge page.

**Comment by supermassimo** [comment]
Now https://www.curseforge.com/minecraft/mc-mods/dynamictrees/files/4589747
